mm2src/coins/eth.rs

```diff
@@ -1138,6 +1145,17 @@ impl MarketCoinOps for EthCoin {
         )
     }
 
+    fn send_raw_tx_bytes(&self, tx: &[u8]) -> Box<dyn Future<Item = String, Error = String> + Send> {
+        let bytes = try_fus!(hex::decode(tx));
```

Comment:
The following test fails with `eth:1149] Odd number of digits`:
```rust
#[test]
fn test_send_raw_tx_bytes() {
    let (_, coin) = eth_coin_for_test(EthCoinType::Eth, vec!["http://eth1.cipig.net:8555".into()], None);
    let tx_bytes = hex::decode("f8a975843b9aca0083024f8394fab46e002bbf0b4509813474841e0716e673013680b84440c10f190000000000000000000000003ef8b4a81ab3444864377dde648268f00e3cd0700000000000000000000000000000000000000000000000004563918244f4000029a0ee799246e00354e173c2236aac52dca3d9e75ac98d2ac48ce67fdab42712c82ca06c42b9db9ebf22fa2aeb85927ba8275ea057f5bffdc2d4bd923606415a18b58a").unwrap();
    coin.send_raw_tx_bytes(&tx_bytes).wait().unwrap();
}
```

While it should fail with `eth:1155] RPC error: Error { code: ServerError(-32000), message: \"invalid sender\", data: None`. Raw bytes of the transaction are submitted here, so `hex::decode` shouldn't be used.

I've also noticed that `trait Transaction::tx_hex` method naming might be confusing. Please feel free to rename it to `raw_tx_bytes` or just `tx_bytes`.

Reply:
I think let's do that renaming in another PR or something. The diffs in this PR already looks messed up. It might be hard to review for you, @shamardy and @sergeyboyko0791

Reply:
Sure, this renaming is not mandatory for this PR, thanks!